client/gutenberg/extensions/map/add-point/style.scss

```diff
@@ -0,0 +1,53 @@
+/** @format */
+
+.component__add-point {
+	position: absolute;
+	left: 50%;
+	top: 50%;
+	z-index: 1;
+	width: 32px;
+	height: 38px;
+	margin-top: -19px;
+	margin-left: -16px;
+	background-image: url( https://s3.amazonaws.com/atavist-static/map/oval.svg );
```

Comment:
We can probably copy these over to this folder and they'll end up in the right place with the SDK?

Reply:
Oops, missed this asset. Done: https://github.com/Automattic/wp-calypso/pull/28498/commits/a3e3c318ad14555e0c6f4f1974434a9076c83ead